model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java

```diff
@@ -36,6 +36,10 @@
     private Date submitDate = new Date();
     private long totalReceivers;
 
+    private long appOpenCounter;
+    private Date firstOpenDate;
+    private Date lastOpenDate;
```

Comment:
we need to integrate this into the migration PR, that @qmx is currently working on for getting ready to be part of master